mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = async (req, res) => {
+  try {
+    const deletionResult = await Order.findOneAndDelete({ _id: req.params.orderId });
+    if (!deletionResult) {
+      return res.status(404).json({ error: "Order not found" });
+    }
+    res.json({ message: "Order removed successfully" });
```

Comment:
The response message "Order removed successfully" should be consistent throughout the application. In the `exports.removeOrder` function, the success message is "Order removed successfully", but in the `exports.productById` function, the success message is "Product deleted successfully". It would be better to define a constant for success messages in a separate file and import it wherever needed to maintain consistency.